spec/abci++/abci++_methods_002_draft.md

```diff
@@ -145,15 +144,40 @@ title: Methods
 
     * Technically optional - not involved in processing blocks.
     * Guardian of the mempool: every node runs `CheckTx` before letting a
-    transaction into its local mempool.
+      transaction into its local mempool.
     * The transaction may come from an external user or another node
     * `CheckTx` validates the transaction against the current state of the application,
-    for example, checking signatures and account balances, but does not apply any
-    of the state changes described in the transaction.
-    not running code in a virtual machine.
-    * Transactions where `ResponseCheckTx.Code != 0` will be rejected - they will not be broadcast to
-    other nodes or included in a proposal block.
-    * Tendermint attributes no other value to the response code
+      for example, checking signatures and account balances, but does not apply any
+      of the state changes described in the transaction.
+    * Transactions where `ResponseCheckTx.Code != 0` will be rejected - they will not be broadcast
+      to other nodes or included in a proposal block.
+      Tendermint attributes no other value to the response code.
+
+### Commit
+
+#### Parameters and Types
+
+* **Request**:
+
+    | Name   | Type  | Description | Field Number |
```

Comment:
I suppose we could omit the table since we do state that the request is empty anyway. (It'd be easy enough to add one later if we add arguments)